locales/zh_CN/LC_MESSAGES/utilities.po

```diff
@@ -107,64 +111,82 @@ msgid ""
 "codegen_kernel_statements                   2820 ->  2789    -1.1%\n"
 "codegen_evaluator_statements                   0 ->    14    +inf%"
 msgstr ""
+"linalg__________________polar_decomp______________________________\n"
+"codegen_offloaded_tasks                       37 ->    39    +5.4%\n"
+"codegen_statements                          3179 ->  3162    -0.5%\n"
+"codegen_kernel_statements                   2819 ->  2788    -1.1%\n"
+"codegen_evaluator_statements                   0 ->    14    +inf%\n"
+"\n"
+"linalg__________________init_matrix_from_vectors__________________\n"
+"codegen_offloaded_tasks                       37 ->    39    +5.4%\n"
+"codegen_statements                          3180 ->  3163    -0.5%\n"
+"codegen_kernel_statements                   2820 ->  2789    -1.1%\n"
+"codegen_evaluator_statements                   0 ->    14    +inf%"
 
 #: ../../utilities.rst:53
 msgid ""
 "Currently ``ti benchmark`` only support benchmarking number-of-"
-"statements, no time benchmarking is included since it depends on hardware"
-" performance and therefore hard to compare if the baseline is from "
-"another machine. We are to purchase a fixed-performance machine as a time"
-" benchmark server at some point. Discussion at: https://github.com"
-"/taichi-dev/taichi/issue/948"
+"statements, no time benchmarking is included since it depends on "
+"hardware performance and therefore hard to compare if the baseline is "
+"from another machine. We are to purchase a fixed-performance machine as "
+"a time benchmark server at some point. Discussion at: https://github.com/"
+"taichi-dev/taichi/issue/948"
 msgstr ""
+"目前， ``ti benchmark`` 仅支持基准测试语句数，不包含时间基准测试，因为它"
+"取决于硬件性能，因此如果基准来自另一台计算机则很难进行比较。 我们将在某个"
+"时间点购买固定性能的机器作为时间基准服务器。 讨论在: https://github.com/"
+"taichi-dev/taichi/issue/948"
 
 #: ../../utilities.rst:58
 msgid ""
 "The suggested workflow for **the PR author** to run the regression tests "
 "is:"
-msgstr ""
+msgstr "建议 **the PR author** 运行回归测试的工作流程是："
```

Comment:
Not translated? Btw, this part is recently updated, so keep this untranslated before `make merge` is OK.

Reply:
Don't know how I missed that lol ...
Well noted :-)